recipes-wpe/wpeframework/wpeframework-clientlibraries_git.bbappend

```diff
@@ -1,6 +1,6 @@
 WPE_COMPOSITOR_CLIENT = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'compositorclient_wayland', 'compositorclient_rpi', d)}"
 
-PACKAGECONFIG_append = " displayinfo playerinfo deviceinfo" 
-PACKAGECONFIG_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', 'vc4graphics', '', d)}"
+PACKAGECONFIG_append = " displayinfo playerinfo deviceinfo cryptography cryptography_openssl"
```

Comment:
I believe .bbappend in this meta layer should only contains defaults and patches. The configuration should be moved to the .conf files. The .conf file should be automatically added to the image.

Could you please move all non default PACKAGECONFIGs to the .conf file?

Reply:
done